docs/faq.rst

```diff
@@ -0,0 +1,225 @@
+*******************************************************************************
+Frequently asked questions
+*******************************************************************************
+
+This topic provides answers to common and frequently asked questions from new HIP
+users and users familiar with NVIDIA CUDA.
+
+HIP Support
+===========
+
+What NVIDIA CUDA features does HIP support?
+-------------------------------------------
+
+The :doc:`NVIDIA CUDA runtime API supported by HIP<hipify:tables/CUDA_Runtime_API_functions_supported_by_HIP>`
+and :doc:`NVIDIA CUDA driver API suupported by HIP<hipify:tables/CUDA_Driver_API_functions_supported_by_HIP>`
+pages describe which NVIDIA CUDA APIs are supported and what the equivalents are.
+The :doc:`HIP API documentation <doxygen/html/index.html>` describes each API and
+its limitations, if any, compared with the equivalent CUDA API.
+
+The kernel language features are documented in the
+:doc:`/reference/cpp_language_extensions` page.
+
+What libraries does HIP provide?
+--------------------------------
+
+HIP provides key math and AI libraries. The full list can be found at
+:doc:`rocm:reference/api-libraries`
+
+What hardware does HIP support?
+-------------------------------
+
+HIP supports AMD and NVIDIA GPUs. See the
+:ref:`prerequisites of the install guide<install_prerequisites>` for detailed
+information.
+
+What operating systems does HIP support?
+----------------------------------------
+
+The supported operating systems are listed in the
+:doc:`rocm:compatibility/compatibility-matrix`.
+
+CUDA and OpenCL
+===============
+
+Is HIP a drop-in replacement for CUDA?
+--------------------------------------
+
+The `HIPIFY <https://github.com/ROCm/HIPIFY>`_ tools can automatically convert
+almost all runtime code. Most device code needs no additional conversion since
+HIP and CUDA have similar names for math and built-in functions. HIP code
+provides the same performance as native CUDA code, plus the benefits of running
```

Comment:
Can we say the "same" performance?
"similar" might be a better term.

Reply:
Changed it to "similar" - I am fairly certain, that it provides the same performance, as all functions are just wrappers around CUDA (that was the whole idea behind HIP), but you can never be sure that there are exotic cases, where the generatede code is not the exact same as if it was written in CUDA.